tests/IntegrationTests/IntegrationTestService/Controllers/WeatherForecastController.cs

```diff
@@ -14,7 +14,7 @@
 namespace IntegrationTestService.Controllers
 {
     [ApiController]
-    [Authorize]
+    [Authorize(AuthenticationSchemes = TestConstants.CustomJwtScheme)]
```

Comment:
Didn't we want to keep the default scheme? given the other controller has another scheme?